html/index.html

```diff
@@ -0,0 +1,207 @@
+					<p>Unobtrusive - no changes should be required to existing project management systems (issue trackers, etc.), owners/managers of projects may participate as much or as little as desired.</p>
+					<p>Pseudononymous - we are after intelligence, and obtaining that info often requires a degree of respect for privacy.</p>
+				</div>
+			</div>
+		</section>
+		<!-- //compare -->
+		<!-- testimonial -->
+		<section id="testimonial" class="pt-5 pb-5 bg-dark">
+			<div class="container">
+				<div class="section-heading pt-5 pb-5 text-center">
+					<h3 class="display-4 text-secondary">What Developers Say</h3>
+				</div>
+				<div class="section-content pb-5 row">
+					<div class="col-md-4 d-md-flex">
+						<blockquote class="text-center">
+							<img class="testimonial-photo"	src="https://via.placeholder.com/80" alt="">
+							<p class="bg-white pt-3 pb-3">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Perferendis neque similique esse quod, quas, ducimus voluptatibus eius sunt facere voluptates tempore nemo reprehenderit quam inventore, dolore.</p>
+							<p class="testimonial-name mb-0 text-secondary">Mike Bhand</p>
+							<p class="testimonial-role text-secondary"><small>CEO, Graphicslab</small></p>
+						</blockquote>
+					</div>
+					<div class="col-md-4 d-md-flex">
+						<blockquote class="text-center">
+							<img class="testimonial-photo"	src="https://via.placeholder.com/80" alt="">
+							<p class="bg-white pt-3 pb-3">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Perferendis neque similique esse quod, quas, ducimus voluptatibus eius sunt facere voluptates tempore nemo reprehenderit quam inventore, dolore.</p>
+							<p class="testimonial-name mb-0 text-secondary">Angelina Doe</p>
+							<p class="testimonial-role text-secondary"><small>Marketing, Graphicslab</small></p>
+						</blockquote>
+					</div>
+					<div class="col-md-4 d-md-flex">
+						<blockquote class="text-center">
+							<img class="testimonial-photo"	src="https://via.placeholder.com/80" alt="">
+							<p class="bg-white pt-3 pb-3">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Perferendis neque similique esse quod, quas, ducimus voluptatibus eius sunt facere voluptates tempore nemo reprehenderit quam inventore, dolore.</p>
+							<p class="testimonial-name mb-0 text-secondary">Mike Bhand</p>
+							<p class="testimonial-role text-secondary"><small>CEO, Graphicslab</small></p>
+						</blockquote>
+					</div>
+				</div>
+			</div>
+		</section>
+		<!-- //testimonial -->
+		<!-- contribute -->
+		<section id="contribute" class="pt-5 pb-5">
+			<div class="container">
+				<div class="section-heading pt-5 pb-5 text-center">
+					<a href="#" class="btn btn-lg btn-dark"><h3 class="display-4">Contribute to Bugmark</h3></a>
+				</div>
+			</div>
+		</section>
+		<!-- //contribute -->
+		<!-- call-to-action -->
```

Comment:
We could leave this call-to-action just for the newsletter and show information about privacy policies, terms and conditions in the footer.